__tests__/pages/collectionspage.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render } from "@testing-library/react";
+import { axe } from "jest-axe";
+import React from "react";
+import { mockCollectionsResponse } from "__tests__/__mocks__/data/mockApiResponses";
+import { CollectionsPage } from "@/src/components/pages/collectionsPage/collectionsPage";
+import CollectionSearchParams from "@/src/types/CollectionSearchParams";
+
+jest.mock("next/navigation", () => ({
+  useRouter: jest.fn(),
+  usePathname: jest.fn(),
+}));
+
+describe.skip("Homepage Accessibility", () => {
```

Comment:
Nit: update the title to CollectionsPage and remove the skip. You may need to mock the `push` function from `useRouter` for the test to run.

Reply:
will deal with this in another PR

Reply:
** i think this is in scope of this PR but if you want to move it out can you link a ticket

Reply:
That code ^ is not working for me for some reason. I was going to spend more time on this when working on https://newyorkpubliclibrary.atlassian.net/browse/DR-3323

Reply:
this does not run ? 

```
import { render } from "@testing-library/react";
import { axe } from "jest-axe";
import React from "react";
import { mockCollectionsResponse } from "__tests__/__mocks__/data/mockApiResponses";
import { CollectionsPage } from "@/src/components/pages/collectionsPage/collectionsPage";
import { useRouter } from "next/navigation";

jest.mock("next/navigation", () => ({
  useRouter: jest.fn(),
  usePathname: jest.fn(),
}));

beforeEach(() => {
  (useRouter as jest.Mock).mockImplementation(() => ({
    pathname: "/collections",
  }))
});

describe("Collections page Accessibility", () => {
  const searchParams = {
    collection_keywords: "flower",
    sort: "title-asc",
    page: "2",
  };
  it("passes axe accessibility test", async () => {
    const { container } = render(
      <CollectionsPage
        data={mockCollectionsResponse}
        params={searchParams}
        renderCollections={true}
      />
    );
    expect(await axe(container)).toHaveNoViolations();
  }, 60000);
});

```
Nbd but because this test exposes a couple other issues that you should resolve in this PR, i think it's worth getting it going

Reply:
This runs, I was putting the `beforeEach(()` on the wrong line.